configs/projects/_shared-pe-bolt-server_with_ruby.rb

```diff
@@ -68,6 +68,7 @@
   proj.setting(:gem_install, "#{proj.host_gem} install --no-rdoc --no-ri --local --bindir=#{proj.bindir}")
 end
 
+proj.setting(:gem_cleanup, "#{proj.host_gem} cleanup")
```

Comment:
I dont understand this change. Also, it seems like we are missing this for installer runtime.